packages/app-desktop/gui/NoteEditor/NoteBody/TinyMCE/TinyMCE.tsx

```diff
@@ -598,6 +598,7 @@ const TinyMCE = (props: NoteBodyEditorProps, ref: any) => {
 			];
 
 			const editors = await (window as any).tinymce.init({
+				content_style: `* {font-size: ${props.fontSize}px; font-family: "${props.fontFamily}"; line-height: 1.5;}`,
```

Comment:
Why the line height?

Reply:
For some reason, the lines overlay each other when the font size increases, So the line-height is going to prevent the overlay.
![overlay](https://github.com/laurent22/joplin/assets/113056556/5be17b3f-adde-4fda-a6f7-f22c5730ba03)

Reply:
Ok but a fixed size of 1.5 means the lines are going to be too tall for small fonts and still too small for large fonts. How is that solved in the Markdown editor or viewer?

Reply:
The original line-height for `body` in the Rich Text Editor is actually `1.6em`:

![image](https://github.com/laurent22/joplin/assets/5626656/145ed408-0a14-4fdd-bff9-43bdabb95d1f)

so I think you need to set it to `1.6em` here also to keep the same look.

Reply:
Yes, I think you are right

Reply:
And what if the font family is not set in the settings, what default does it use?

Reply:
The 1.6em is actually not a static value; it comes from the themeStyle. So, I used the same value in the TinyMCE.

Reply:
@laurent22 
I will investigate whether it is replaced or overridden.
If the font family is not set in the settings, it takes the default font family, the same as what the code mirror does.

Reply:
@laurent22 
- The `1.6em` in the `body` is not applied to the text in the editor because the TinyMCE editor creates an `iframe` to isolate the editor's content from the rest of the application. This `iframe` contains its own HTML document with its own style, so the `line-height` of the main body doesn't affect the elements in the `iframe`.

![CSS](https://github.com/laurent22/joplin/assets/113056556/7691d16f-2f0e-48a1-bcb9-ea816aa688ff)


An alternate solution would be to give the `body` of the `iframe` (#tinymce) a `line-height: inherit;`


https://github.com/laurent22/joplin/assets/113056556/33417850-bca3-4afb-bf93-3a7f118c50da



- The content_style doesn't replace the style; it is injected into the head of the page.
https://www.tiny.cloud/docs/configure/content-appearance/#content_style

Reply:
That's great, thank you for looking into this. Indeed if it's an iframe it's not going to get parent style.